pkg/agentfs/docker.go

```diff
@@ -0,0 +1,265 @@
+package agentfs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/charmbracelet/huh"
+
+	"github.com/livekit/protocol/logger"
+)
+
+const (
+	pythonDockerfileURL   = "https://raw.githubusercontent.com/livekit-examples/agent-deployment/refs/heads/main/python-agent-example-app/Dockerfile"
+	pythonDockerIgnoreURL = "https://raw.githubusercontent.com/livekit-examples/agent-deployment/refs/heads/main/python-agent-example-app/.dockerignore"
+	pythonEntrypoint      = "main.py"
+	nodeDockerfileURL     = "https://raw.githubusercontent.com/livekit-examples/agent-deployment/refs/heads/main/node-agent-example-docker/Dockerfile"
+	nodeDockerIgnoreURL   = "https://raw.githubusercontent.com/livekit-examples/agent-deployment/refs/heads/main/node-agent-example-docker/.dockerignore"
```

Comment:
are we confident these aren't going to change? should we load them from a config endpoint in the api?

Reply:
Yeah, that's reasonable, it will probably be helpful to have a generic config endpoint for a few different things anyways, will add this in.